test/arrays.coffee

```diff
@@ -287,3 +287,21 @@ test "for-from comprehensions over Array", ->
 
   array2 = (a + b for [a, b] from [[10, 20], [30, 40], [50, 60]] when a + b >= 70)
   ok array2.join(' ') is '70 110'
+
```

Comment:
@zdenko a couple other ideas for tests:
- check that eg `f .5` -> `f(.5)` (as suggested by @connec)
- some longer accessor chains eg `a.1.2.3.4.5`
- some tests with `@` eg `@1`, `@1.'a'` (note that eg `@'b'` compiles into a tagged template expression so there's a slight inconsistency)